src/main/java/hellos/Hellos.java

```diff
@@ -34,9 +34,8 @@ public String generateOutput() {
     builder.append(natashaSaysHello());
     builder.append(nicSaysHello());
     builder.append(richardSaysHello());
-    builder.append(thomasSaysHello());
-
-
+    builder.append(thomasSaysHello())
```

Comment:
```suggestion
    builder.append(thomasSaysHello());
```

Oops, looks like you've got a missing semicolon 🙃